includes/functions.inc.php

```diff
@@ -292,4 +292,39 @@ function cygwinConversion($path) {
 	}
 	return $path;
 }
+
+/**
+ * Helper function to define custom autoloader 
+ * @param string $rootPath
+ * @param string $prefix
+ * @param string $class
+ * 
+ * @return void
+ */
+function customAutoload($rootPath, $prefix, $class) {
+	if (substr($class, 0, strlen($prefix)) !== $prefix) {
+		return;
+	}
+
+	$class = substr($class, strlen($prefix));
+	$parts = explode('\\', $class);
+
+	// we expect at least one folder in the namespace
+	// there is no class defined directly under classes/ folder
+	if (count($parts) < 2) {
+		return;
+	}
+
+	$className = Core::cleanFileVar(array_pop($parts));
+	$parts = array_map(function($part) {
+		return strtolower(Core::cleanFileVar($part));
```

Comment:
I don't think we can universally `strtolower` these -- looking e.g. at `lib/pkp/classes/stageAssignment`. I think everything is `lowerCamelCase`.